### ISSUE TITLE
Hiding HS labels also hides App Library labels, even if the option for App Library is off.

### ISSUE THREAD

**Issue by sevenpastzeero** [ren7995/Atria#3]
^title

Also, any plans to switch from Aldris colorpicker to iOS 14 native color picker?

**Comment by ren7995** [comment]
Can you show an image of the problem?

**Comment by sevenpastzeero** [comment]
![IMG_3266](https://user-images.githubusercontent.com/34801304/118813929-4cfbdd80-b8b8-11eb-81e1-8cdbebb9217c.PNG)
![IMG_3267](https://user-images.githubusercontent.com/34801304/118813945-5127fb00-b8b8-11eb-96de-7cab4b30a322.PNG)
![IMG_3268](https://user-images.githubusercontent.com/34801304/118813955-54bb8200-b8b8-11eb-8055-2839680d4d70.PNG)
![IMG_3269](https://user-images.githubusercontent.com/34801304/118813969-57b67280-b8b8-11eb-87b0-7b89c3b0fd56.PNG)

I have only Hide HS Icons Labels option enabled, and Hide App Library Labels off. The HS Labels are hidden, but as you can see, App Library Labels are hidden too, even though the option is off. Is it a bug? or am I doing something wrong?
Thank you.

**Comment by ren7995** [comment]
Inside of app library folders is intentionally made hidden when the home screen icon labels are, but if you respring the other labels should appear (for the category folders)

**Comment by sevenpastzeero** [comment]
For some reason it is still not showing the app library categories labels. I tried turning the option on the respring, then turning it off and respring, but no luck.

**Comment by ren7995** [comment]
I might've changed it so HS labels applies to all labels (including app library)... are you sure you don't have another tweak like snowboard that disables them?

**Comment by sevenpastzeero** [comment]
No, I don't have any other tweak that modify HS.

Also, I found anothet bug regarding hiding labels in folders and app library. I took a vid capture.

**Comment by ren7995** [comment]
Where did you send it?

**Comment by sevenpastzeero** [comment]
https://user-images.githubusercontent.com/34801304/119225808-c8ef6300-bb0e-11eb-9b7c-fd95b024dcc7.MP4
 Here it is. I didn't k ow if I should post it here or as a new bug.
If you have Hide Lables in Folders off, you can see that random apps labels appears in app library folders.

**Comment by ren7995** [comment]
I can't seem to reproduce this. It's available in the source, there isn't really any way for it to break - I would have to imagine there's another tweak causing a conflict

**Comment by sevenpastzeero** [comment]
I did some testing. The 2nd bug happens only if you open a folder residing one screen before app library (which in my case because I have only 1 HS page). If I create a second hs page with apps only then the bug does not occur. But if I put a folder in 2nd page and open it then switch to app library, then the bug happen.

Original bug still there even when I turned off all my other tweaks with icleaner.
